src/Symfony/Component/Routing/Loader/AnnotationClassLoader.php

```diff
@@ -94,7 +94,7 @@ public function setRouteAnnotationClass($class)
      *
      * @throws \InvalidArgumentException When route can't be parsed
      */
-    public function load($class, $type = null)
+    public function load($class, string $type = null)
```

Comment:
This will break compatibility with Config 4.4. Please either revert this change or update Routing‘s composer.json accordingly.